exercises/anagram/canonical-data.json

```diff
@@ -21,6 +21,20 @@
       },
       "expected": ["stream", "maters"]
     },
+    {
+      "uuid": "03eb9bbe-8906-4ea0-84fa-ffe711b52c8b",
+      "reimplements": "b3cca662-f50a-489e-ae10-ab8290a09bdc",
+      "comments": [
+        "Reimplemented to be consistent about removing references to 'master'"
+      ],
+      "description": "detects two anagrams",
+      "property": "findAnagrams",
+      "input": {
+        "subject": "solemn",
+        "candidates": ["lemons", "pigeon", "melons"]
```

Comment:
Other ideas for non-anagrams: model, novel, pleon

Reply:
"pigeon" is there because I was trying to keep it as close to the test case that it `reimplements` as possible. That is: three six-letter candidates, where the first and last are anagrams. I should've written that, sorry - I've edited the top post to show that existing test case.

But now that I'm looking at "pigeon" again: previously, it had only one letter in common with the `subject`, but now it has three. So, depending on how strictly we interpret `reimplements`, I think we should change "pigeon" to another six-letter word that only has one letter in common with "solemn". Perhaps "parrot" or "puffin", to stay in the same genre.

This test case reimplemented by this PR is the second one, so I think it's supposed to avoid testing edge cases. This exercise should definitely test the kind of near-anagrams that you suggest, but from a quick look at the `canonical-data.json` I think we already do that in these two cases:
https://github.com/exercism/problem-specifications/blob/ef9e1e17c84721ee6e9d5a65c8dd3ba2122eac91/exercises/anagram/canonical-data.json#L24-L43

And in this test case:
https://github.com/exercism/problem-specifications/blob/ef9e1e17c84721ee6e9d5a65c8dd3ba2122eac91/exercises/anagram/canonical-data.json#L121-L130

Any thoughts?

Reply:
I forgot the S for the sixth letter, they are all plurals. Parrot is fine.
Another: Timber

Reply:
Maybe `cherry`?

Extreme bikeshedding, but I didn't love that:
- "parrot" is also a verb, so "lemons parrot melons" can be read strangely.
- With the input `["lemons", "parrot", "melons"]` (or with `"timber"`) producing `["lemons", "melons"]`, the function could be a "fruit detector function" rather than an anagram detector. Although with  `["lemons", "cherry", "melons"]` it could be a plural detector...

Reply:
Then, we could pick a plural fruit: olives, grapes, apples